moveit_kinematics/ikfast_kinematics_plugin/README.md

```diff
@@ -0,0 +1,9 @@
+# MoveIt! IKFast
+
+Authors: Dave Coleman, CU Boulder; Jeremy Zoss, SwRI; David Butterworth, KAIST, Mathias Lüdtke, Fraunhofer IPA
+
+Generates a IKFast kinematics plugin for MoveIt using [OpenRave](http://openrave.org/) generated cpp files.
+
+Tested on ROS Hydro and greater with Catkin using OpenRave 0.8 with a 6dof and 7dof robot arm manipulator. Does not work with greater than 7dof.
```

Comment:
I know this PR is about rearranging packages but I'm not sure that this line still makes sense to keep as it has to have been tested for this code to make it into the main development branch.